src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/Permissions.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using OrchardCore.Search.Elasticsearch.Core.Services;
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Search.Elasticsearch
+{
+    public class Permissions : IPermissionProvider
+    {
+        private readonly ElasticIndexSettingsService _elasticIndexSettingsService;
+
+        public static readonly Permission ManageIndexes = new Permission("ManageIndexes", "Manage Indexes");
```

Comment:
And a similar distinction with Lucene permissions too, for clarity.

```suggestion
        public static readonly Permission ManageElasticIndexes = new Permission("ManageElasticIndexes ", "Manage Elasticsearch Indexes");
```